static/test/javascripts-legacy/helpers/svg-paths.js

```diff
@@ -0,0 +1,45 @@
+define(function() {
```

Comment:
Can do! Although it would be _even better_ to add the following comment:

```js
// projects/common/svgs.js
// DEPRECATED: please don't use this file to inject SVGs into your markup
// Instead load the SVG directly into your JavaScript module
```